opendebates/utils.py

```diff
@@ -131,6 +131,8 @@ def sort_list(citations_only, sort, ideas):
         ideas = ideas.order_by("votes")
     elif sort == "-local_votes":
         ideas = ideas.order_by("-local_votes")
+    elif sort == "-current_votes":
```

Comment:
Is it worth checking whether the sitemode state is currently allowing this sort order, to prevent people from manually constructing a query string with it when the sitemode option is unset?  (Most of time that result would just be uninteresting, so maybe not worth the extra logic.)